master/buildbot/steps/shell.py

```diff
@@ -248,7 +248,7 @@ def buildCommandKwargs(self, warnings):
 
         # check for the usePTY flag
         if 'usePTY' in kwargs and kwargs['usePTY'] is not None:
-            if self.workerVersionIsOlderThan("svn", "2.7"):
+            if self.workerVersionIsOlderThan("shell", "2.7"):
```

Comment:
This shall not change anything as buildbot worker advertise versions for '*', just fix it for coherency